spellcheck/config/Makefile

```
@@ -0,0 +1,6 @@
+DICTIONARIES=$(shell find . -name '*.txt')
+
+lint:
```

```suggestion
sort:
```
Name it after what it does